scipy/stats/_continuous_distns.py

```diff
@@ -5160,7 +5160,12 @@ def _rvs(self):
 
     def _pdf(self, x):
         # maxwell.pdf(x) = sqrt(2/pi)x**2 * exp(-x**2/2)
-        return np.sqrt(2.0/np.pi)*x*x*np.exp(-x*x/2.0)
+        # The value 0.7978845608028654 is sqrt(2/pi).
```

Comment:
How about making a constant out of this?
```
SQRT_2_OVER_PI = 0.7978845608028654
```
and then
```
    def _pdf(self, x):
         # maxwell.pdf(x) = sqrt(2/pi)x**2 * exp(-x**2/2)
        return SQRT_2_OVER_PI*x*x*np.exp(-x*x/2.0)
```

Reply:
Sure.  Using the literal is a bit faster than using the name, since the name requires a lookup, but either is much better than evaluating `np.sqrt(2.0/np.pi)` in every call.

Reply:
Switched to predefined constants in the last commit.